frontend/src/components/VMediaInfo/VMediaTags.vue

```diff
@@ -1,25 +1,45 @@
 <template>
   <ul v-if="tags.length" class="flex flex-wrap gap-2">
+    <div v-if="additionalSearchViews">
+      <VTag
+        v-for="(tag, index) in tags"
```

Comment:
```suggestion
        v-if="additionalSearchViews"
        v-for="(tag, index) in tags"
```
You don't need to wrap the tags in a separate `div`. Also, we need to hide the `VMediaTag`s if additionalSearchViews is true.